keps/sig-release/2572-release-cadence/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: Defining the Kubernetes Release Cadence
+kep-number: 2572
+authors:
+  - "@justaugustus"
+owning-sig: sig-release
+participating-sigs:
+  - sig-architecture
+  - sig-testing
+status: implementable
+creation-date: 2021-01-21
+reviewers:
+  - "@BenTheElder"
+  - "@derekwaynecarr"
+  - "@dims"
+  - "@hasheddan"
+  - "@jeremyrickard"
+  - "@johnbelamaric"
+  - "@spiffxp"
+  - "@stevekuznetsov"
```

Comment:
```suggestion
  - "@stevekuznetsov"
  - "@ehashman"
```

Reply:
Added you in the most recent commit but up in sorted order. Thanks for the feedback ❤️